AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/config/sofar/sources_id_metadata.json

```diff
@@ -601,5 +576,86 @@
     "water_depth": 31,
     "water_depth_units": "m",
     "wave_buoy_type": "directional"
+  },
+ "SPOT-31574C":
+  {
+    "spotter_id": "SPOT-31574C",
+    "site_name": "Hillarys",
+    "latitude_nominal": -31.85125,
+    "longitude_nominal": 115.6464667,
+    "institution": "University of Western Australia",
```

Comment:
In this instance `institution` should be "IMOS Coastal Wave Buoys" and `institution_code` "UWA". This is because the UWA deployed a spotter from another account( and thus API access token) for an IMOS deployment. It is related to the PR https://github.com/aodn/data-services/pull/1218 you merged today

Reply:
Changed institution accordingly, thanks again for reviewing those changes.